modules/ding_event/ding_event.info

```diff
@@ -123,5 +121,5 @@ features[variable][] = webform_node_ding_event
 features[views_view][] = ding_event
 features_exclude[field_base][field_place2book_tickets] = field_place2book_tickets
 features_exclude[field_instance][node-ding_event-field_place2book_tickets] = node-ding_event-field_place2book_tickets
+features_exclude[variable][node_options_ding_event] = node_options_ding_event
```

Comment:
Why have this been exclude?

Reply:
The "promote types to frontpage" uses this, and we don't want these settings to be overriden when the feature is reverted

Reply:
What is "promote types to frontpage"?